benches/write.rs

```diff
@@ -0,0 +1,63 @@
+#[cfg(feature = "serde")] // To prevent clippy from yelling :)
```

Comment:
Nope, I spent a bit more time to make read benchmarks work with and without the serde flags, you can do it too